includes/definitions/discovery/moxa-etherdevice.yaml

```diff
@@ -14,3 +18,61 @@ modules:
                     descr: 'Power Consumption'
                     index: 'powerConsumption.{{ $index }}'
                     value: powerConsumption
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::poePortConsumption
```

Comment:
This should just be `oid: poePortConsumption`

Add MOXA-EDSP510A8POE-MIB at the top like:

```
mib: MOXA-EDSP510A8POE-MIB
```

Reply:
goes for all the others in this file.